apps/web/app/[locale]/auth/error/page.tsx

```diff
@@ -1,30 +1,30 @@
 'use client';
 
+import { useSearchParams } from 'next/navigation';
 import SadCry from '@components/ui/svgs/sad-cry';
 import { Text } from 'lib/components';
-import { useTranslations } from 'next-intl';
 import Link from 'next/link';
-/**
- * Error page
- *
- * @description the page that will be shown if any social login failed. This is not related to Next.js error file
- * @returns a custom component that shows error
- */
-export default function Page() {
-	const t = useTranslations();
-	return (
+import UnauthorizedPage from '@components/pages/unauthorized';
+
+enum Error {
+	Configuration = 'Configuration',
+	AccessDenied = 'AccessDenied'
+}
+
+const errorMap = {
+	[Error.Configuration]: (
 		<div className="mt-28 flex flex-col gap-7 items-center">
 			<div className="m-auto relative flex justify-center items-center gap-4 text-center ">
 				<SadCry width={97} height={97} />
-				<Text className="text-[78px] font-semibold text-chetwodeBlue">{t('pages.error.TITLE')}</Text>
+				<Text className="text-[78px] font-semibold text-chetwodeBlue">Error !</Text>
 			</div>
 
 			<Text className="text-[40px] font-bold text-center text-[#282048] dark:text-light--theme">
-				{t('pages.error.HEADING_TITLE')}
+				Something went wrong !
```

Comment:
@GloireMutaliko21 why you remove translations!?

Reply:
Ohhh ! Will fix it in next PR...